code/modules/projectiles/guns/ds13/flamethrower.dm

```diff
@@ -468,6 +468,7 @@
 
 /obj/item/weapon/reagent_containers/glass/fuel_tank/hydrazine
 	name = "fuel tank (hydrazine)"
+	vacuum_burn = TRUE
```

Comment:
this shouldn't be necessary as the fuel updating proc handles it. If there's a situation where its not being called, find that and put in a call